theme/templates/resource-landing-page/modals.html

```diff
@@ -740,6 +740,54 @@ <h4 class="modal-title" id="add-contributor"> Add Contributor </h4></div>
     </div>
 </div>
 
+<div class="modal fade in" id="extraMetaDialog" tabindex="-1" role="dialog"
+     aria-labelledby="add-extra-meta" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <form action="/hsapi/_internal/{{ cm.short_id }}/contributor/add-metadata/"
```

Comment:
@pkdash Is this form action correct here? Why does edit extra metadata model submit triggers ```contributor/add-metadata``` action?